go.mod

```diff
@@ -2,7 +2,7 @@ module github.com/grafana/mimir
 
 go 1.21.0
 
-toolchain go1.23.0
+toolchain go1.22.6
```

Comment:
There is go 1.22.5 in the build-image: https://github.com/grafana/mimir/blob/c727e59acf6e03b10978c14313e2cf5a75c369f2/mimir-build-image/Dockerfile#L8

Reply:
Done.